src/repr/src/row.rs

```diff
@@ -2826,12 +2834,10 @@ impl Default for RowArena {
 ///
 /// [`SharedRow::get`] panics when trying to obtain multiple references to the shared row.
 #[derive(Debug)]
-pub struct SharedRow(Rc<RefCell<Row>>);
+pub struct SharedRow(RefCell<Row>);
 
 impl SharedRow {
-    thread_local! {
-        static SHARED_ROW: Rc<RefCell<Row>> = Rc::new(RefCell::new(Row::default()));
-    }
+    const SHARED_ROW: RefCell<Row> = RefCell::new(Row::empty());
```

Comment:
other option would be the following:

```rust
thread_local! {
  static SHARED_ROW: RefCell<Row> = const { RefCell::new(Row::empty()) }
}
```

Reply:
from thread_local documentation:

```
/// This macro supports a special `const {}` syntax that can be used
/// when the initialization expression can be evaluated as a constant.
/// This can enable a more efficient thread local implementation that
/// can avoid lazy initialization. For types that do not
/// [need to be dropped][crate::mem::needs_drop], this can enable an
/// even more efficient implementation that does not need to
/// track any additional state.
```